lua/indent_blankline/utils.lua

```diff
@@ -68,6 +69,9 @@ M.is_indent_blankline_enabled =
         if g_enabled ~= true then
             return false
         end
+        if respect_list then
+          return vim.opt.list:get()
```

Comment:
This should not overwrite the other checks.
if it is `false`, return `false`, if it is `true`, continue with the other checks.

And this function is memoized, you have to get the value in `init.lua` and pass it as an argument